src/test/java/com/jcabi/dynamo/mock/H2DataTest.java

```diff
@@ -134,7 +134,26 @@ public void createsTablesWithLongNames() throws Exception {
      */
     @Test
     public void supportsTableNamesWithIllegalCharacters() throws Exception {
-        new H2Data().with(".-.", new String[] {"pk"}, new String[0]);
+        new H2Data().with(".-.", new String[]{"pk"}, new String[0]);
+    }
+
+    /**
+     * H2Data supports column names with characters illegal to H2.
+     * todo: #28 user reports that H2Data does not support
+     * COLUMNS with ".", "-", or digits but i don't know for sure
```

Comment:
todo should state what needs to be done, here if someone takes this todo he/she won't know what do do here.
You should ask issue creator for details.